.github/workflows/linux_riscv_xuantie_dev_cpu.yml

```diff
@@ -2,13 +2,20 @@ name: Linux RISC-V with Xuantie (Ubuntu 22.04, Python 3.10)
 on:
   workflow_dispatch:
     inputs:
-      testFilter:
-        description: 'Filter for google tests'
+      testFilterType:
+        description: 'Filter for google tests {SMOKE | CONCISE}'
         required: true
-        default: '*smoke_AdaPoolAvg4DLayoutTest*'
+        default: 'CONCISE'
   pull_request:
     paths:
```

Comment:
We cannot use paths filter for required GitHub checks (and this one is required) due to [GitHub limitations](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks) (see also https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/ci/github_actions/smart_ci.md#skipping-an-entire-workflow-for-specific-changes). Either we remove ci/gha_overall_status_linux_riscv from required (so it won't block merge / won't participate in merge queue checks); or we handle it via [Smart CI](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/ci/github_actions/smart_ci.md#smart-ci-overview) instead.